apps/tenant-management-webapp-e2e/src/integration/events/events.page.ts

```diff
@@ -126,10 +126,113 @@ class eventsPage {
       `//*[@data-testid="Platform-stream-table"]//tbody/tr/td[text()="${streamName}"]/following-sibling::td//button[@data-testid="toggle-stream-visibility"]`
     );
   }
+  streamTableDetails(streamName) {
```

Comment:
This one is a duplicated page object of streamDetails.

Reply:
This one object works for both tables tenant and core